Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -412,6 +412,21 @@ public let CodeGenerators: [CodeGenerator] = [
         b.endIf()
     },
 
+    CodeGenerator("SwitchCaseGenerator", input: .anything) { b, cond in
+        b.beginSwitch(on: cond) { cases in
+            cases.addDefault {
+                b.generateRecursive()
+            }
+            for _ in 0..<Int.random(in: 0...5) {
+                var fallsThrough = false
```

Comment:
Can this just be `let fallsThrough = probability(0.1)`?

Reply:
Or I guess even just
`cases.add(b.randVar(), fallsThrough: probability(0.1)) {`
below